<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MQTTErrorWidget</class>
 <widget class="QWidget" name="MQTTErrorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MQTT Error Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lUserName">
     <property name="text">
      <string>User name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lPassword">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lePassword"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leUserName"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="bChange">
     <property name="text">
      <string>Make Changes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lErrorType">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lId">
     <property name="text">
      <string>Client ID:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leId"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
